apps/docs/pages/guides/cli/managing-environments.mdx

```diff
@@ -251,21 +255,22 @@ on:
 
 jobs:
   deploy:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
We should probably pin the version as we don't want a non LTS version which might have breaking changes break the CI/CD pipeline.

Reply:
That's very unlikely to happen as the cli binary is agnostic to ubuntu versions, ie. we don't ship separate binaries for 20.04 or 22.04.

Reply:
Also our [CI test](https://github.com/supabase/cli/blob/main/.github/workflows/ci.yml#L49) is using `ubuntu-latest` so if something really breaks, we will be the first to know.